addons/event/models/event_registration.py

```diff
@@ -47,6 +47,9 @@ class EventRegistration(models.Model):
         ('draft', 'Unconfirmed'), ('cancel', 'Cancelled'),
         ('open', 'Confirmed'), ('done', 'Attended')],
         string='Status', default='draft', readonly=True, copy=False, tracking=True)
+    # event related field added for search
```

Comment:
I would move those 2 under "event_end_date", sine those fields are already related to the event_id, and are under the "organisation" category (which fits well with organizer and responsible).

Reply:
Thanks, indeed, I have also removed my comment; other fields are not commented and have probably the same purpose